src/modules/UI/scenes/SendConfirmation/sendConfirmationOptions.test.js

```diff
@@ -0,0 +1,77 @@
+/* globals jest describe it expect */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+
+import SendConfirmationOptions from './sendConfirmationOptions.js'
+
+describe('SendConfirmation', () => {
+  it('should render with standard props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      changeMiningFee: jest.fn,
+      openHelpModal: jest.fn,
+      sendMaxSpend: jest.fn,
+      uniqueIdentifierModalActivated: jest.fn,
+      sourceWallet: {},
+      currencyCode: 'BTC',
+      isEditable: true
+    }
+    const actual = renderer.render(<SendConfirmationOptions {...props} />)
+
+    expect(actual).toMatchSnapshot()
+  })
+
+  it('not editable', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      changeMiningFee: jest.fn,
+      openHelpModal: jest.fn,
+      sendMaxSpend: jest.fn,
+      uniqueIdentifierModalActivated: jest.fn,
+      sourceWallet: {},
+      currencyCode: 'BTC',
+      isEditable: false
+    }
+    const actual = renderer.render(<SendConfirmationOptions {...props} />)
+
+    expect(actual).toMatchSnapshot()
+  })
+
+  it('XMR', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      changeMiningFee: jest.fn,
+      openHelpModal: jest.fn,
+      sendMaxSpend: jest.fn,
```

Comment:
Does `sendMaxSpend` need to be removed from this test for XMR? @wolverineks

Reply:
i pass it in for each currency, but if the currencyCode === 'XMR' then the button shouldnt show